components/lib/fetch-httpurlconnection/src/test/java/mozilla/components/lib/fetch/httpurlconnection/HttpUrlConnectionFetchTestCases.kt

```diff
@@ -34,4 +38,18 @@ class HttpUrlConnectionFetchTestCases : mozilla.components.tooling.fetch.tests.F
         connection.setupWith((Request("https://mozilla.org", useCaches = false)))
         assertFalse(connection.useCaches)
     }
+
+    @Test
+    override fun getDataUri() {
+        // We can't run this test case because Base64 encoding is not available but we can verify
+        // that we attempt to fetch the data URI.
```

Comment:
If we run this with `AndroidJUnit4` then we should be able to use the Base64 classes, I think?

Reply:
Ah yes, good catch! I've upgraded the lib and removed the test override as the base test is running fine now!